src/analysis/rust_type.rs

```diff
@@ -146,29 +193,24 @@ pub fn parameter_rust_type(env: &Env, type_id:library::TypeId,
                 match direction {
                     library::ParameterDirection::In |
                         library::ParameterDirection::Return => rust_type,
-                    _ => Err(format!("/*Unimplemented*/{}", rust_type.as_str())),
+                    _ => Err(TypeError::Unimplemented(into_inner(rust_type))),
                 }
             } else {
-                format_parameter(rust_type, direction)
+                rust_type.map_any(|s| format_parameter(s, direction))
             }
         }
         Alias(ref alias) => {
-            let res = format_parameter(rust_type, direction);
-            if parameter_rust_type(env, alias.typ, direction, nullable, ref_mode).is_ok() {
-                res
-            } else {
-                res.and_then(|s| Err(s))
-            }
+            let rust_type = rust_type.map_any(|s| format_parameter(s, direction));
```

Comment:
Now `Atom` shows as `/*Ignored*/gdk::Atom_` (Ex. in `gtk_text_buffer_register_serialize_tagset`)
Seems this not equal change.